docs/WOMtool.md

```diff
@@ -11,7 +11,7 @@ The following is the toolchain used for development of womtool.  Other versions
 
 ## Building
 
-`sbt assembly` will build a runnable JAR in `target/scala-2.11/`
+`sbt assembly` will build a runnable JAR in `womtool/target/scala-2.11/`
```

Comment:
Not `2.12`?

Reply:
Oh sure we can update to whatever is the latest now.